config/src/wasm.rs

```diff
@@ -3,7 +3,7 @@
 use iroha_config_base::derive::{Documented, Proxy};
 use serde::{Deserialize, Serialize};
 
-const DEFAULT_FUEL_LIMIT: u64 = 20_000_000;
+const DEFAULT_FUEL_LIMIT: u64 = 25_000_000;
```

Comment:
So we do more computations on the WASM side without extra communication with the host. That's why it increased?

Reply:
yes, it should reduce with my next PR where I would reduce computation on wasm side